docs/nais-application/migration.md

```diff
@@ -0,0 +1,192 @@
+# Migrate from GHCR to GAR
+
+## Introduction
+
+This document describes how to migrate from GHCR (GitHub Container Registry) to GAR (Google Artifact Registry).
+
+## Why migrate?
+
+GHCR serves as a container registry hosted by GitHub, but it has certain limitations, such as imposing strict rate
+limits on the number of requests. We've encountered challenges with these rate limits, particularly when upgrading a
+cluster or restoring from a backup. In such instances, we've had to patiently wait for the rate limits to reset before
+deploying / restoring our applications.
+
+On the other hand, GAR is a container registry hosted by Google, and we find it to be a more favorable solution for our
+needs. Additionally, as we transition to the cloud, the seamless integration with the Google Cloud Platform further
+enhances its appeal as a beneficial option for our operations.
+
+### Migrate
+
+If you currently have a workflow utilizing GHCR, transitioning to GAR can be achieved by updating the workflow with the
+following steps, to illustrate the migration process, let's consider an example of a workflow that currently relies on
+GHCR. We will use the example as a case study to demonstrate how to migrate to GAR.
```

Comment:
```suggestion
You can switch from GHCR to GAR by following these steps
to update your current workflow. Let's simplify this by taking an
example of a workflow using GHCR and show you exactly how to change it
to GAR.
```

or something like that, This section has pretty fancy words and a passive voice, idk how we tone-of-voice that.